test/media-segment-request.test.js

```diff
@@ -1171,3 +1050,377 @@ QUnit.skip('id3 callback does not fire if partial data has no ID3 tags', functio
+      id3: 0,
+      videoSegmentTimingInfo: 1
+    }, 'calls as expeceted');
+    done();
+  };
+
+  mediaSegmentRequest(this.options);
+
+  assert.equal(this.requests[0].uri, 'foo.ts', 'segment-request');
+  this.standardXHRResponse(this.requests[0], muxedSegment());
+});
+
+QUnit.test('video ts segments will make it to the transmuxer', function(assert) {
+  const done = assert.async();
+
+  this.options.segment.transmuxer = this.createTransmuxer();
+  this.options.segment.resolvedUri = 'foo.ts';
+  this.options.doneFn = () => {
+    assert.deepEqual(this.calls, {
+      data: 1,
+      trackInfo: 1,
+      progress: 1,
+      timingInfo: 2,
+      captions: 0,
+      id3: 0,
+      videoSegmentTimingInfo: 1
+    }, 'calls as expeceted');
+    done();
+  };
+
+  mediaSegmentRequest(this.options);
+
+  assert.equal(this.requests[0].uri, 'foo.ts', 'segment-request');
+  this.standardXHRResponse(this.requests[0], videoSegment());
+});
+
+QUnit.test('audio ts segments will make it to the transmuxer', function(assert) {
+  const done = assert.async();
+
+  this.options.segment.transmuxer = this.createTransmuxer();
+  this.options.segment.resolvedUri = 'foo.ts';
+  this.options.doneFn = () => {
+    assert.deepEqual(this.calls, {
+      data: 1,
+      trackInfo: 1,
+      progress: 1,
+      timingInfo: 2,
+      captions: 0,
+      id3: 0,
+      videoSegmentTimingInfo: 0
+    }, 'calls as expeceted');
+    done();
+  };
+
+  mediaSegmentRequest(this.options);
+
+  assert.equal(this.requests[0].uri, 'foo.ts', 'segment-request');
+  this.standardXHRResponse(this.requests[0], audioSegment());
+});
+
+QUnit.test('aac with id3 will make it to the transmuxer', function(assert) {
+  const done = assert.async();
+
+  this.options.segment.transmuxer = this.createTransmuxer();
+  this.options.segment.resolvedUri = 'foo.aac';
+  this.options.doneFn = () => {
+    assert.deepEqual(this.calls, {
+      data: 1,
+      trackInfo: 48,
+      progress: 1,
+      timingInfo: 2,
+      captions: 0,
+      id3: 0,
+      videoSegmentTimingInfo: 0
+    }, 'calls as expeceted');
+    done();
+  };
+
+  mediaSegmentRequest(this.options);
+
+  assert.equal(this.requests[0].uri, 'foo.aac', 'segment-request');
+  this.standardXHRResponse(this.requests[0], aacWithId3Segment());
+});
+
+/** TODO: fix this case in mux.js
+QUnit.test('aac without id3 will make it to the transmuxer', function(assert) {
+  const done = assert.async();
+
+  this.options.segment.transmuxer = this.createTransmuxer();
+  this.options.segment.resolvedUri = 'foo.aac';
+  this.options.doneFn = () => {
+    assert.deepEqual(this.calls, {
+      data: 1,
+      trackInfo: 47,
+      progress: 1,
+      timingInfo: 2,
+      captions: 0,
+      id3: 0,
+      videoSegmentTimingInfo: 0
+    }, 'calls as expeceted');
+    done();
+  };
+
+  mediaSegmentRequest(this.options);
+
+  assert.equal(this.requests[0].uri, 'foo.aac', 'segment-request');
+  this.standardXHRResponse(this.requests[0], aacSegment());
```

Comment:
Should this be something like `aacWithoutId3Segment`? (and same for one further down)